docs/dev/libs/state_processor_api.md

```diff
@@ -413,45 +551,25 @@ The `KeyedStateBootstrapFunction` supports setting event time and processing tim
 The timers will not fire inside the bootstrap function and only become active once restored within a `DataStream` application.
 If a processing time timer is set but the state is not restored until after that time has passed, the timer will fire immediatly upon start.
 
-Once one or more transformations have been created they may be combined into a single `Savepoint`. 
-`Savepoint`'s are created using a state backend and max parallelism, they may contain any number of operators. 
+<span class="label label-danger">Attention</span> If your bootstrap function creates timers, the state can only be restored using one of the [process]({{ site.baseurl }}/dev/stream/operators/process_function.html) type functions.
+
+## Modifying Savepoints
 
-<div class="codetabs" markdown="1">
-<div data-lang="java" markdown="1">
-{% highlight java %}
-Savepoint
-    .create(backend, 128)
-    .withOperator("uid1", transformation1)
-    .withOperator("uid2", transformation2)
-    .write(savepointPath);
-{% endhighlight %}
-</div>
-<div data-lang="scala" markdown="1">
-{% highlight scala %}
-Savepoint
-    .create(backend, 128)
-    .withOperator("uid1", transformation1)
-    .withOperator("uid2", transformation2)
-    .write(savepointPath)
-{% endhighlight %}
-</div>
-</div>
-		
 Besides creating a savepoint from scratch, you can base on off an existing savepoint such as when bootstrapping a single new operator for an existing job.
 
 <div class="codetabs" markdown="1">
 <div data-lang="java" markdown="1">
 {% highlight java %}
 Savepoint
-    .load(backend, oldPath)
+    .load(new MemoryStateBackend(), oldPath)
     .withOperator("uid", transformation)
     .write(newPath);
 {% endhighlight %}
 </div>
 <div data-lang="scala" markdown="1">
 {% highlight scala %}
 Savepoint
-    .load(backend, oldPath)
+    .load(new MemoryStateBackend(), oldPath)
     .withOperator("uid", transformation)
     .write(newPath)
 {% endhighlight %}
```

Comment:
About potential corruption below: I guess, you could delete the new savepoint though. Can you rephrase that?

Reply:
No you cannot delete the new savepoint. It would delete the shared operators.

Reply:
I believe there are three parameters for Savepoint.load(): ExecutionEnvironment, a path and StateBackend. Could you double check? To me, the Savepoint.load() call at the beginning of the "Reading State" section is a good one.

Reply:
Good catch, @qinjunjerry 

@sjwiesman: what I meant is that you can always remove the new savepoint without affecting the old one because the old one is not touched, is it?

Reply:
ok, let's stay cautious here - it depends how you delete the new savepoint: if Flink (in the future) would expose a savepoint delete method, this would then only look at the metadata file and also delete files from the old one, so we keep the sentence as it is